sdk/digitaltwins/.gitignore

```diff
@@ -1,3 +1,4 @@
 *.config.json
```

Comment:
Is there a need to have an overriding .gitignore file in your sub-directories? If not I'd suggest sticking with the one in the root.

Reply:
We've contributed to the repo root .gitignore file. We've added files specific to our solution to this one. I wouldn't feel safe making these exclusions at the root level.

Also, it seems as if our config.json files will be going away with a recent feature added by Pavel.

Reply:
OK if we expect them to go away soon I'm not worried about it but in general we are trying to drive a level of consistency across all the SDK's and we do that by trying to avoid special cases for some libraries so if we it is something interesting for you then I'd like to put it in the root version and only if we need an exceptional case do we override here.